core/src/integration-test/java/com/scalar/db/storage/dynamo/DynamoIntegrationTest.java

```diff
@@ -1,52 +1,20 @@
 package com.scalar.db.storage.dynamo;
```

Comment:
Cause the type of concatenated clustering key changes from string to binary so those tests also need to be changed, so they are related to this PR otherwise the CI will be failed.

Reply:
Yes, it requires the update but most of the code changes are more like the updates that should have been done in the previous PR, I feel.

Reply:
@thongdk8 I think we can apply these integration test changes first in another PR and then we can come back this optimization change. That is not causing any CI failure, correct?

Reply:
@brfrn169 you are right, I'll create another PR for this.

Reply:
@feeblefakie @brfrn169 I created [a PR](https://github.com/scalar-labs/scalardb/pull/318) for this. Please take a look.